modules/010-deckhouse-flow-schema/docs/README.md

```diff
@@ -0,0 +1,3 @@
+---
+title: "The deckhouse-flow-schema module"
+---
```

Comment:
A detailed explanation of why this module was added in the first place is required.

Reply:
yes